app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] @felixodette great work kindly add the foreign key associating the user with posts comments and likes.
`hint`
```suggestion
  has_many :posts, foreign_key: 'author_id'
```